apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -373,6 +390,41 @@ export default class PublishAction extends BaseRushAction {
     }
   }
 
+  private _updateAPIFile(packageName: string, project: RushConfigurationProject): void {
```

Comment:
>_updateAPIFile [](start = 10, length = 14)

Uhhh... this is really strange. Why does Rush have a dependency on api-extractor?  Why is it using a custom config file parser?